docs/admin-ports.md

```diff
@@ -0,0 +1,30 @@
+# Admin Ports
+
+The following is a list of TCP ports used by internal DC/OS services, and their corresponding systemd unit.
+
+## All Roles
+
+ - 1050: dcos-ddt
```

Comment:
I'm hoping to make this doc easy to maintain by sticking to just ports and unit names. Knowing the unit name makes it easy to look up the service if you're curious.